plugins/synapticsmst/fu-plugin-synapticsmst.c

```diff
@@ -152,6 +152,7 @@ fu_plugin_synaptics_add_device (FuPlugin *plugin,
 	dev_id_str = g_strdup_printf ("MST-%s-%s-%u-%u",
 				      kind_str, aux_node, layer, rad);
 	fu_device_set_id (dev, dev_id_str);
+	fu_device_set_platform_id (dev, aux_node);
```

Comment:
Is this really the hardware ID? It's designed to be something based on the physical connection.

Reply:
Yeah the aux node will change if it's physically connected differently.